using-mobile-extensions/adobe-journey-optimizer/README.md

```diff
@@ -80,164 +87,199 @@ Follow these steps to integrate the Adobe Journey Optimizer extension.
+            Identity.registerExtension();
+            Messaging.registerExtension();
+            MobileCore.start(new AdobeCallback() {
+                @Override
+                public void call(final Object o) {
+                    // processing after start
+                }
+            });
+        } catch (Exception e) {
+            // handle the exception
+        }
     }
 }
 ```
 {% endtab %}
 
-{% tab title="iOS" %}
-### Swift
+{% tab title="iOS (AEP 3.x)" %}
+
+#### Swift
 
 ```swift
 // AppDelegate.swift
 func application(_ application: UIApplication, didFinishLaunchingWithOptions launchOptions: [UIApplication.LaunchOptionsKey: Any]?) -> Bool {
     MobileCore.registerExtensions([Identity.self, Edge.self, Messaging.self], {
-        MobileCore.configureWith(appId: "yourLaunchEnvironmentID")
+        MobileCore.configureWith(appId: "YOUR_APP_ID")
     })
-  ...
+
+    ...
 }
 ```
 
-### Objective-C
+#### Objective-C
 
 ```objc
 // AppDelegate.m
-- (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(NSDictionary *)launchOptions {
+- (BOOL) application:(UIApplication *)application didFinishLaunchingWithOptions:(NSDictionary *)launchOptions {
     [AEPMobileCore registerExtensions:@[AEPMobileEdgeIdentity.class, AEPMobileEdge.class, AEPMobileMessaging.class] completion:^{
-    [AEPMobileCore configureWithAppId: @"yourLaunchEnvironmentID"];
-  }];
-  ...
+        [AEPMobileCore configureWithAppId: @"YOUR_APP_ID"];
+    }];
+
+    ...
 }
 ```
+
 {% endtab %}
+
+{% tab title="iOS (ACP 2.x)" %}
+
+This extension is built on the AEPCore (3.x) and it is not compatible with ACPCore (2.x). Please follow [the guide for migrating to the Swift AEPCore](https://aep-sdks.gitbook.io/docs/resources/migrate-to-swift).
+
+{% endtab %}
+
 {% endtabs %}
 
 #### Sync the user push token with Adobe
 
-After importing and registering the extensions with your application, you need to sync the push token with Adobe Experience Platform, by using the setPushIdentifier API.
+After importing and registering the extensions with your application, you need to sync the push token with Adobe Experience Platform, by using the `setPushIdentifier` API.
 
 {% tabs %}
+
 {% tab title="Android" %}
-To retrieve the push token from Firebase Messaging Service, please read the tutorial on [retrieving the registration token](https://firebase.google.com/docs/cloud-messaging/android/client#retrieve-the-current-registration-token) within the Firebase documentation. After retrieving the push token, you can use the following API to sync it with Profile in Platform.
 
-### Java
+To retrieve the push token from Firebase Messaging Service, read the tutorial on [retrieving the registration token](https://firebase.google.com/docs/cloud-messaging/android/client#retrieve-the-current-registration-token) within the Firebase documentation. After retrieving the push token, use the following API to sync it with the Profile in AEP.
```

Comment:
Should be "... the Profile in Platform.", since AEP is against branding guidelines.